### PR TITLE
settingswatcher: address todo for GetStorageClusterVersion

### PR DESCRIPTION
**Pull request by rafiss**
v22 is now minted, so we can use it in this function. On the release-22.2 branch, it's only minted in v22.2.0-rc.1, so we can no longer use v22.2.0-beta.5 for the mixed version tests.

Epic: None

Release note: None

### CODE REVIEW

File: pkg/server/settingswatcher/settings_watcher.go [hunk 1]
```diff
@@ -435,8 +435,7 @@ func (s *SettingsWatcher) GetStorageClusterVersion() clusterversion.ClusterVersi
 		// settingswatcher, it means that the storage cluster has not yet been
 		// upgraded to 23.1. As a result, assume that storage cluster is at
 		// version 22.2.
-		// TODO(ajstorm): change this to 22.2 once the new version is minted.
-		storageClusterVersion := roachpb.Version{Major: 22, Minor: 1, Internal: 76}
+		storageClusterVersion := roachpb.Version{Major: 22, Minor: 2, Internal: 0}
```
Comment (ajstorm) [inline pkg/server/settingswatcher/settings_watcher.go:438]: Ultra-nit: you might want to drop Internal altogether.

**Comment by ajstorm** [comment]
LGTM

**Review by ajstorm — APPROVED** [review]

**Review by renatolabs — APPROVED** [review]
Thanks!

File: pkg/server/settingswatcher/settings_watcher.go (hunk 1 above)
Comment (rafiss) [reply pkg/server/settingswatcher/settings_watcher.go:438]: i kind of like making it explicit so it's easier to tie it back to this line: https://github.com/cockroachdb/cockroach/blob/8e9b54896ce88d108da154c9134cab5f5c37625a/pkg/clusterversion/cockroach_versions.go#L522-L523

**Comment by rafiss** [comment]
tftr!

bors r=ajstorm,renatolabs

**Pull request merged**